app/src/main/java/co/techmagic/hr/presentation/time_tracker/TimeTrackerFragment.kt

```diff
@@ -69,20 +78,45 @@ class TimeTrackerFragment : BaseViewFragment<TimeTrackerPresenter>(), TimeTracke
     }
 
     override fun notifyDayReportsChanged(date: Calendar) {
-        daysAdapter.notifyItemChanged(daysAdapter.dateToPage(date))
+        //      daysAdapter.notifyItemChanged(daysAdapter.dateToPage(date))
+        daysAdapter.notifyDataSetChanged()
+        //todo fixme to notify about item changed in inner adapter
```

Comment:
It's perfectly fine calling `daysAdapter.notifyItemChanged(daysAdapter.dateToPage(date))` here. It will cause invoking of bind method only for corresponding day view holder.
To update inner adapter you'd better use DiffUtil in the `setNewData(...)` method of `TimeReportAdapter`.
```fun setNewData(newData: List<UserReportViewModel>) {
        this.data.clear()
        this.data.addAll(newData)
        notifyDataSetChanged() // replace this with a DiffUtil 👌
    }```

Reply:
Done